command/pki_list_children_command.go

```diff
@@ -0,0 +1,263 @@
+package command
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/ryanuber/columnize"
+)
+
+type PKIListChildrenCommand struct {
+	*BaseCommand
+
+	flagConfig          string
+	flagReturnIndicator string
+	flagDefaultDisabled bool
+	flagList            bool
+
+	flagUseNames bool
+
+	flagSignatureMatch    bool
+	flagIndirectSignMatch bool
+	flagKeyIdMatch        bool
+	flagSubjectMatch      bool
+	flagPathMatch         bool
+}
+
+func (c *PKIListChildrenCommand) Synopsis() string {
+	return "Determine Which (of a List) of Certificates Were Issued by A Given Parent Certificate"
+}
+
+func (c *PKIListChildrenCommand) Help() string {
+	helpText := `
+Usage: vault pki list-intermediates PARENT CHILD-LIST
+PARENT is the certificate that might be the issuer that everything should be verified against.
+CHILD-LIST is a comma-separated list of intermediates to be compared to the PARENT.  It can be omitted, in which case a
```

Comment:
Wonder if it might be more ergonomic to make it be `vault pki list-intermediates PARENT [CHILD [CHILD ...]]` i.e., take zero or more children as additional arguments, doing auto-detection otherwise? I don't _think_ comma is a valid path character though, but just thinking it'd be harder to do the comma variant (you'd probably end up with a subshell `$(... | tr '\n' ',')` or something).

Reply:
comma isn't a valid path character, but I've tried writing in this...